src/class-scripts.php

```diff
@@ -137,7 +137,7 @@ public function enqueue_js_tracker(): void {
 	 */
 	public function script_loader_tag( string $tag, string $handle, string $src ): string {
 		$parsely_options = $this->parsely->get_options();
-		if ( in_array(
+		if ( \in_array(
```

Comment:
Do you remember what kind of warning this resolves?

Reply:
By forcing to use the global namespace, it enables some opcode optimizations. 

- https://veewee.github.io/blog/optimizing-php-performance-by-fq-function-calls/
- https://github.com/cakephp/cakephp/issues/12654
- [PHP master showing specially handled namespace functions](https://github.com/php/php-src/blob/0ae6a6755071c42964d979a81e7800819c9d2261/Zend/zend_compile.c#L4300-L4364)

There are a couple of tools around that will allow this to be added at compile time.